### PR TITLE
Add Vue to the scaffolding app

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
A couple of things:

* I assume `neutrino-preset-web` will be included in `neutrino-preset-vue`. To that reason, I did not add `neutrino-preset-web` to the user's dev-dependencies.
* `neutrino-preset-vue` is running on old neutrino. There is a release candidate version out there, however the scaffolding app is simply done to run the latest stable package.

File: packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<script>
+  export default {
+    name: 'App',
+    data() {
+      return {};
+    }
+  };
+</script>
+
+<template>
```
Comment (eliperelman) [inline packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/App.vue]: Change indentation to 2 spaces?

File: packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <div :class="$style.App">
+        Welcome to <%= data.directory %>
+    </div>
+</template>
+
+<style module>
+    .App {
+        padding: 20px;
+    }
+</style>
```
Comment (eliperelman) [inline packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/App.vue]: Add blank line to end of file.

File: packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
+import App from './App.vue';
+
```
Comment (eliperelman) [inline packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/index.js]: Is there anything we can do here for HMR?

Comment (helfi92) [reply packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/index.js]: `vue-loader` has HMR integrated within haha. https://vue-loader.vuejs.org/en/features/hot-reload.html

There's no need to add `if (module.hot)...`. The `<template>, <script>, <style>` in vue files does it out of the box when you have `vue-loader`.

Comment (eliperelman) [reply packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/index.js]: 😍 😍 😍 😍 😍 😍 😍 😍

**Pull request merged**